task/ecosystem-cert-preflight-checks/0.2/ecosystem-cert-preflight-checks.yaml

```diff
@@ -213,6 +219,40 @@ spec:
           value: "$(steps.generate-container-auth.results.auth-json-path)"
         - name: PFLT_KONFLUX
           value: "true"
+        - name: PARAM_IMAGE_URL
+          value: $(params.image-url)
+        - name: PARAM_PLATFORM
+          value: $(params.platform)
+      script: |
+        #!/usr/bin/env bash
+
+        set -o errexit
+        set -o nounset
+        set -o pipefail
+
+        image_url="${PARAM_IMAGE_URL}"
+        platform="${PARAM_PLATFORM}"
+
+        if [ -n "$platform" ]; then
+          # Extract part after slash if present
+          arch="${platform#*/}"
+          if [ "$arch" = "x86_64" ]; then
+            arch="amd64"
+          fi
+
+          # Validate against supported arch list. If it's not a known arch, fallback to amd64
+          case "$arch" in
+            amd64|ppc64le|arm64|s390x)
+              ;;
+            *)
+              arch="amd64"
+              ;;
+          esac
```

Comment:
@mshaposhnik , do you expect that this pattern will be followed? Should we document the assumption of this pattern somewhere in the future to ensure that we keep track of the dependency?